cmake/install.cmake

```diff
@@ -87,22 +90,35 @@ if(NOT MSVC)
 else()
   set(CMAKE_INSTALL_CMAKEDIR "cmake" CACHE STRING "${_cmakedir_desc}")
 endif()
+mark_as_advanced(CMAKE_INSTALL_CMAKEDIR)
```

Comment:
if debug_postifx & CMAKE_INSTALL_PREFIX aren't advanced variables, this shouldn't be either.